Frontend/src/Components/PPC_MPPT/PPC_MPPT.js

```diff
@@ -1,28 +1,32 @@
-import {Flex, Spacer} from "@chakra-ui/react";
+import {Flex, GridItem, Spacer, Grid, VStack, Text, useColorMode} from "@chakra-ui/react";
 import PPC_CELL from "./PPC_CELL";
 import MPPT_CELL from "./MPPT_CELL";
 import OutputCurrent from "./OutputCurrent";
 import HeadingCell from "../Shared/HeadingCell";
-import RangeCell from "../Shared/RangeCell";
 import CONSTANTS from "../../data-constants.json";
+import RangeBar from "../Shared/RangeBar";
+import getColor from "../Shared/colors";
 
 
 export default function PPC_MPPT(props) {
+    const { colorMode } = useColorMode();
+    const headerBg = getColor("header", colorMode);
+
     return (
-        <Flex flex='auto' direction='column'>
+        <Flex flex='auto' direction='column' overflowY='scroll'>
             <HeadingCell fontSize='2.2vh' label='Power Path Controller'/>
```

Comment:
```suggestion
            <HeadingCell fontSize='2vh' label='Power Path Controller'/>
```
To keep it consistent with the other HeadingCell, we should change this one's fontSize to 2vh as well